cluster-operator/src/main/java/io/strimzi/operator/cluster/model/VolumeUtils.java

```diff
@@ -405,4 +407,90 @@ private static String getVolumeNameHashStub(String name)   {
+
+            if (trustedCertificates != null && trustedCertificates.size() > 0) {
+                for (CertSecretSource certSecretSource : trustedCertificates) {
+                    addNewVolume(volumeList, certSecretSource, isOpenShift, alias);
+                }
+            }
+        }
+    }
+
+    /**
+     * Creates the Volumes used for authentication of Kafka client based components, checking that the named volume has not already been
+     * created.
+     */
+    private static void addNewVolume(List<Volume> volumeList, CertSecretSource certSecretSource, boolean isOpenShift, String alias) {
+        String volumeName = alias != null ? alias + '-' + certSecretSource.getSecretName() : certSecretSource.getSecretName();
+        // skipping if a volume with same name was already added
+        if (!volumeList.stream().anyMatch(v -> v.getName().equals(volumeName))) {
+            volumeList.add(VolumeUtils.createSecretVolume(volumeName, certSecretSource.getSecretName(), isOpenShift));
+        }
+    }
+
+    /**
+     * Creates the Client Volume Mounts
+     *
+     * @param tls    Kafka Client tls object
+     * @param volumeMountList    List where the volume mounts will be added
+     * @param tlsCertBasedVolumeMount   Path where the TLS certs should be mounted
+     */
+    protected static void createClientVolumeMounts(ClientTls tls, List<VolumeMount> volumeMountList, String tlsCertBasedVolumeMount) {
+        createClientVolumeMounts(tls, volumeMountList, tlsCertBasedVolumeMount, null);
+    }
+
+    /**
+     * Creates the Client Volume Mounts
+     *
+     * @param tls    Kafka Client tls object
+     * @param volumeMountList    List where the volume mounts will be added
+     * @param tlsCertBasedVolumeMount   Path where the TLS certs should be mounted
+     * @param alias   Alias to reference the Kafka Cluster
+     */
+    protected static void createClientVolumeMounts(ClientTls tls, List<VolumeMount> volumeMountList, String tlsCertBasedVolumeMount, String alias) {
+
+        if (tls != null) {
+            List<CertSecretSource> trustedCertificates = tls.getTrustedCertificates();
+
+            if (trustedCertificates != null && trustedCertificates.size() > 0) {
+                for (CertSecretSource certSecretSource : trustedCertificates) {
+                    String volumeMountName = alias != null ? alias + '-' + certSecretSource.getSecretName() : certSecretSource.getSecretName();
+                    addNewVolumeMounts(volumeMountList, volumeMountName, tlsCertBasedVolumeMount, certSecretSource);
+                }
+            }
+        }
+    }
+
+    /**
+     * Creates the VolumeMounts used for authentication of Kafka client based components, checking that the named volume mount has not already been
+     * created.
+     */
+    protected static void addNewVolumeMounts(List<VolumeMount> volumeMountList, String volumeMountName, String tlsCertBasedVolumeMount, CertSecretSource certSecretSource) {
```

Comment:
Describe the parameters in Javadoc. Also, does it add volume mounts? Or volume mount? (plural versus singular.)